package.json

```
@@ -7,27 +7,30 @@
     "type": "git",
     "url": "git://github.com/flumedb/flumeview-query.git"
   },
+  "scripts": {
+    "test": "tape test/*.js",
```

this change needed to run travis windows
necessitated changes to how test folder was structured (some test were util + tests)

---

If this is still failing on Windows, try using single-quotes!

```suggestion
    "test": "tape 'test/*.js'",
```